src/web/services/account.service.spec.ts

```diff
@@ -42,13 +42,22 @@ describe('AccountService', () => {
   });
 
   it('should execute POST on account endpoint', () => {
+    const testKey: string = 'testKey';
+    const paramMap: Record<string, string> = {
+      key: testKey,
+    };
+    service.createAccount(testKey);
+    expect(spyHttpRequestService.post).toHaveBeenCalledWith(ResourceEndpoints.ACCOUNT, paramMap);
+  });
+
+  it('should execute POST on account request endpoint', () => {
```

Comment:
Can I check this point again. The test case is checking if the endpoint is called successfully when the method is called. In my opinion, adding the conditions to the description will not accurately describe the test being carried out here (since the conditions are not checked in the test at all).

Reply:
`createAccount` and `createAccountRequest` are very similar, yet very different. Other endpoints have very clear cut difference, thus why just describing "should execute ... on ..." suffices.
Then again it should be fine to keep the description as brief as possible.